tests/integration_test/test_output.py

```diff
@@ -0,0 +1,179 @@
+from tests.integration_test.solutions import isomorphism_solution, automorphism_solution
+from tests.integration_test.settings import problem, console_pass, write_to_csv
+from tests.integration_test.result_output import result_output
+from input_output.sys_output import fail, passed, title
+from input_output.file_output import create_csv_file, write_csv_line
+
+
+def test_output(filename, n_graphs, processing_time, iso_test_result, iso_count_test_result):
+    # Get results from solution file
+    iso_solution = []
+    count_solution = {}
+    if filename in isomorphism_solution:
+        iso_solution = isomorphism_solution[filename]
+    if filename in automorphism_solution:
+        count_solution = automorphism_solution[filename]
+
+    # Dependent on the solutions that are available, print test results or not
+    solution_exists = True
+    if problem == 1:
+        if not bool(iso_solution):
+            solution_exists = False
+    if problem == 2:
+        if not bool(iso_solution) and not bool(count_solution):
+            solution_exists = False
+    if problem == 3:
+        if not bool(count_solution):
+            solution_exists = False
+
+    if not solution_exists:
+        result_output(filename, n_graphs, processing_time, iso_test_result, iso_count_test_result)
+        return -1
```

Comment:
Waarom return je hier -1? Dan wordt de error_count in run_algorithm eentje later.

Reply:
oh inderdaad! Woeps!

Reply:
hier ben ik ook benieuwd naar